js/packages/web/src/views/home/index.tsx

```diff
@@ -47,14 +55,21 @@ export const HomeView = () => {
   const liveAuctions = auctions
   .sort((a, b) => a.auction.info.endedAt?.sub(b.auction.info.endedAt || new BN(0)).toNumber() || 0);
 
+  const items =
+    activeKey === LiveAuctionViewState.All
+      ? liveAuctions
```

Comment:
This is great. I'm merging this. I'd love to see a follow up where we included in aprticipated those auctions that also have your wallet as m.auctionManager.info.authority.equals(wallet.publicKey) too (you made the auction).